Automattic/wc-calypso-bridge
# Add "Powered by WooCommerce" footer

This PR adds a "Powered by WooCommerce" footer to all WooCommerce Calypso pages, to match Store on WP.com.

Fixes #171.

<img width="711" alt="screen shot 2018-11-15 at 12 35 03 pm" src="https://user-images.githubusercontent.com/689165/48570981-aba10480-e8d3-11e8-983f-c7fc1b6b7651.png">

To Test:
* Load up a WooCommerce page and verify the footer is shown.
* Load up a setup page (`/wp-admin/admin.php?page=wc-setup`) and verify the footer is not shown.
